build_custom_app.py

```diff
@@ -413,7 +413,7 @@ def upload_apk(self, apk_file):
 
     def publish_release(self, options, versionCodes):
         return self.make_request('tracks', 'update',
-            track="alpha",
+            track="internal",
```

Comment:
Why this change ?
What is the purpose of `internal` ?
I had set the track to `alpha` to be able to "promote" the `alpha` version to `beta` and so (publish the release, ...)
Is it possible with `internal` ?

Reply:
Internal is a level below alpha and can be promoted to alpha beta or production. It rolls out instantly to internal testers unlike alpha which can take 3-4 hours.